beacon-chain/db/kv/state.go

```diff
@@ -323,6 +330,61 @@ func (s *Store) saveStatesEfficientInternal(ctx context.Context, tx *bolt.Tx, bl
 	return s.storeValidatorEntriesSeparately(ctx, tx, validatorsEntries)
 }
 
+func getPhase0PbState(rawState interface{}) (*ethpb.BeaconState, error) {
```

Comment:
Had to do these to fix `cognitive complexity 80 of func `(*Store).saveStatesEfficientInternal` is high (> 65)`